contents/tutorials/identifying-users-guide.md

```diff
@@ -0,0 +1,67 @@
+---
+title: An introductory guide to identifying users in PostHog
+sidebar: Docs
+showTitle: true
+author: ['ian-vanagas']
+date: 2022-11-21
+featuredImage: ../images/tutorials/banners/identifying-users-guide.png
+topics: ["configuration"]
```

Comment:
Per your recent PR, what topics should we add here?

Reply:
good call, needs persons